doc/MANUAL.adoc

```diff
@@ -931,6 +932,27 @@ Optional attributes:
 * *update-mtime*: If *true*, update the modification time (mtime) of cache
   entries that are read. The default is *false*.
 
+=== HTTP storage backend
+
+URL format: `http://HOST:PORT/PATH`
+
+This backend stores data in an HTTP compatible server. The required HTTP
+methods are `GET`, `PUT`, and `DELETE`.
+
+Note that ccache will not perform any cleanup of the HTTP storage.
+
+Examples:
+
+* `http://localhost:8080/`
+* `http://example.org/cache`
+
+Known issues and limitations:
+
+* The cpp-httplib cannot handle URLs containing IPv6 addresses like: `http://[::1]/`
```

Comment:
```suggestion
* URLs containing IPv6 addresses like `http://[::1]/` are not supported
```

Reply:
Did you mean to apply this or did you not agree with my suggestion?

I don't think that we should mention cpp-httplib here since it's an implementation detail that only will confuse the reader.

Reply:
I simply overlooked it. I'm not too used to GitHub PR comments. At work we use BitBucket which handles force-pushes much better.